app/org/sagebionetworks/bridge/models/accounts/Withdrawal.java

```diff
@@ -0,0 +1,35 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import org.joda.time.DateTime;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Request to withdraw from the current study.
+ *
+ */
+public final class Withdrawal {
```

Comment:
I find it a little unusual that serializing this would give you JSON that you can't deserialize back into a Withdrawal class. I feel like it would be cleaner if the Withdrawal object only contains the reason (as strictly speaking, that's the only user input into the withdraw API), and have the controller pass in the withdrewOn as a separate parameter.